docusaurus/docs/Android/02-client/06-guides/06-offline-support.mdx

```diff
@@ -0,0 +1,62 @@
+# Offline Support
+
+The offline library provides a caching mechanism applied automatically to some of the `ChatClient` functions as side effects.
+The offline results, as well as online results, are being exposed through a `StateFlow` objects that can be accessed using state related objects.
+
+When using the offline library, you first have to provide the `OfflinePlugin` to `ChatClient` as described on the [Getting Started](../../01-basics/03-getting-started.mdx#adding-an-offline-plugin) page.
+
+### Configuration
+
```

Comment:
Maybe add the same snippet here, as users might not have seen the getting started page (or skipped)